docs/src/pages/components/DataTable.svx

```diff
@@ -192,6 +192,73 @@ The slot name for the table header cells is `"cell-header"`.
   ]}"
 />
 
+### With custom display and sort methods
+
+<DataTable sortable title="Load balancers" description="Your organization's active load balancers."
+  headers="{[
+    { key: "name", value: "Name" },
+    { key: "protocol", value: "Protocol" },
+    { key: "port", value: "Port" },
+    { key: "cost", value: "Cost", display: (cost) => cost + " €" },
+    {
+      key: "expireDate",
+      value: "Expire date",
+      display: (date) => new Date(date).toLocaleString(),
+      sort: (a, b) => (new Date(a) <= new Date(b) ? -1 : 1),
```

Comment:
The date comparison operation may be simplified with `new Date(a) - new Date(b)` for better results.

fixed in #382